test/test_torch.py

```diff
@@ -525,7 +525,7 @@ def test_csub(self):
     @staticmethod
     def _test_neg(self, cast):
         float_types = ['torch.DoubleTensor', 'torch.FloatTensor', 'torch.LongTensor']
-        int_types = ['torch.IntTensor', 'torch.ShortTensor']
```

Comment:
test for CharTensor?  I don't know if there's an easy way from python to figure out the what the behavior should be.

Reply:
Done